src/main/java/com/powsybl/openloadflow/dc/DcLoadFlowEngine.java

```diff
@@ -62,16 +62,8 @@ private static double getActivePowerMismatch(Collection<LfBus> buses) {
 
     @Override
     public DcLoadFlowResult run() {
-        EquationSystem<DcVariableType, DcEquationType> equationSystem = context.getEquationSystem();
-
-        boolean succeeded = false;
-        try (JacobianMatrix<DcVariableType, DcEquationType> j = context.getJacobianMatrix()) {
```

Comment:
@geofjamg and @obrix I have been obliged to delete the exception catching, but I am still not sure that it is the good fix. What do you think?

Reply:
This is ok